src/core/core.scale.js

```diff
@@ -138,9 +140,11 @@ function parseFontOptions(options, nestedOpts) {
 		fontFamily: valueOrDefault(nestedOpts.fontFamily, options.fontFamily),
 		fontSize: valueOrDefault(nestedOpts.fontSize, options.fontSize),
 		fontStyle: valueOrDefault(nestedOpts.fontStyle, options.fontStyle),
-		lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight)
+		lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight),
```

Comment:
It shouldn't be necessary to add a trailing comma to this line

Reply:
removed